src/main/scala/io/qbeast/spark/table/IndexedTable.scala

```diff
@@ -30,13 +30,26 @@ trait IndexedTable {
    */
   def exists: Boolean
 
+  /**
+   * Returns whether the table is converted to Qbeast format.
+   * @return
+   */
+  def isConverted: Boolean
```

Comment:
IMHO this name could be better. The problem with "Converted" is that it seems that the table had a different format, but later was converted to Qbeast. What if the table was Qbeast from the very beginning. What question does the method tries to answer?

Reply:
It tries to answer if this particular table contains Qbeast Metadata of any type (staging, revision..). 

If the precondition of creating an `IndexedTable` is that this Table is already indexed (or converted), I will remove the condition and add checks before initializing the particular IndexedTable.

Reply:
Then I would suggest to name the method as `hasQbeastMetadata` or alike.

Reply:
Done!